src/ValueFormatters/QuantityHtmlFormatter.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace ValueFormatters;
+
+/**
+ * HTML formatter for quantity values.
+ *
+ * @since 0.5.1
+ *
+ * @licence GNU GPL v2+
+ * @author Thiemo Mättig
+ */
+class QuantityHtmlFormatter extends QuantityFormatter {
+
+	/**
+	 * @param string $number
+	 * @param string $unit
+	 *
+	 * @return string HTML
+	 */
+	protected function applyUnit( $number, $unit ) {
+		$html = htmlspecialchars( $number );
```

Comment:
So, if applyUnit() is not called (because self::OPT_APPLY_UNIT is false, or no unit is given), this formatter would return $number unescaped? Not good, who knows what it may contain after localization...